generic_expression_patterns_modules/ranking.py

```diff
@@ -35,7 +37,9 @@ def concat_simulated_data(local_dir, num_runs, project_id, data_type):
         Number of simulated experiments
     project_id: str
         Project id to use to retrieve simulated experiments
-    data_type: 'DE' or 'GSEA'
+    data_type: 'DE' or 'GSA'
```

Comment:
Does 'GSA' here just mean some kind of "gene set analysis" (i.e. enrichment analysis)?

Reply:
Yea. So "GSA" refers to gene set analysis where as "GSEA" is a particular method. At the time GSEA was the only enrichment method I was using so that was why I had it hard coded. But now that there is the possibility of using other methods I wanted to update the name